### ISSUE TITLE
[RTL] ColumnSorting

### ISSUE THREAD

**Issue by Siemienik** [handsontable/handsontable#8813]
### Description 

For RTL, the sorting icon should be displayed on the left.

![image](https://user-images.githubusercontent.com/2893099/135806400-7c1713f2-a0c2-4e0f-a243-a416cf2b0eef.png)

### How to reproduce 

1. Use this [configuration](https://github.com/Siemienik/handsontable-storybooks/blob/2fcf1af3e9a7bfd98329dfa8ec6e9984f0e44b5a/vanilla/configurations.js#L684-L703):
```
{
    data: [
        ['Tesla', 'Model 3', 'BlueStar', 'USA', '★★★★'],
        ['Tesla', 'Model S', 'WhiteStar', 'USA', '★★★★★'],
        ['Mitsubishi', 'iMiEV', '', 'Japan', '★★'],
        ['Ford', 'Focus EV', '', 'USA', '★★'],
        ['Mitsubishi', 'iMiEV Sport', '', 'Japan', '★★'],
        ['Tesla', 'Roadster', 'DarkStar', 'USA', '★★★★★'],
        ['Volkswagen', 'e-Golf','', 'Germany', '★★'],
        ['Volkswagen', 'E-Up!', '', 'Germany', '★★'],
        ['Ford', 'C-Max Energi', '', 'USA', '★'],
        ['BYD', 'Denza', '', 'China', '★★★'],
        ['BYD', 'e5', '', 'China', '★★★'],
        ['BYD', 'e6', '', 'China', '★★★★']
    ],
    colHeaders: ['Brand', 'Model', 'Code name', 'Country of origin', 'Rank'],
    columnSorting: true,
    height: 'auto',
    licenseKey: 'non-commercial-and-evaluation'
}
```

**Comment by Siemienik** [comment]
As long as it will be the little update I will do this issue with #8928

**Comment by Siemienik** [comment]
done in #9102

**Comment by aninde** [comment]
Fixed with v12.0.0 https://jsfiddle.net/ke0wq5af/

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 (RTL Support), published in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
